src/nvim/tui/tui.c

```diff
@@ -131,7 +131,7 @@ static void terminfo_start(UI *ui)
   data->can_use_terminal_scroll = true;
   data->bufpos = 0;
   data->bufsize = sizeof(data->buf) - CNORM_COMMAND_MAX_SIZE;
-  data->showing_mode = 0;
+  data->showing_shape = SHAPE_IDX_N;
```

Comment:
Also why does github believe that responses to comments are part of a "pending review"?